.circleci/config.yml

```diff
@@ -124,6 +127,30 @@ jobs:
           paths:
             - "*"
 
+  build_x86_linux_cxx17:
+    docker:
+      - image: buildpack-deps:bionic
```

Comment:
If this build is merely meant to check for forwards compatibility, we could probably change this to ``buildpack-deps:disco``, which should contain cvc4 1.6 - that way we could keep cvc4 enabled below.

Reply:
Can you add a commit testing that assumption? I'm fine with that, should enable us to ensure C++17 compatibility in the SMT subsystem.

Reply:
Let's see what happens!

Reply:
Looks like it works (SMT stuff just compiled on the CI run).

Reply:
You want to approve the PR then? 😉